DATA/production/configurations/asyncReco/setenv_extra.sh

```diff
@@ -3,7 +3,9 @@
 
 # process flags passed to the script
 
-export SETENV_NO_ULIMIT=1
+if [[ -z "$ALIEN_JDL_USEGPUS" && $ALIEN_JDL_USEGPUS != 1 ]]; then
```

Comment:
I think this should have been || instead of &&

Reply:
Correct, see https://github.com/AliceO2Group/O2DPG/pull/1063